ouroboros-consensus-test/src/Test/Util/Serialisation/Roundtrip.hs

```diff
@@ -323,10 +326,13 @@ roundtrip_SerialiseNodeToClient
   => CodecConfig blk
   -> [TestTree]
 roundtrip_SerialiseNodeToClient ccfg =
-    [ rt (Proxy @blk)                    "blk"
-    , rt (Proxy @(GenTx blk))            "GenTx"
-    , rt (Proxy @(ApplyTxErr blk))       "ApplyTxErr"
-    , rt (Proxy @(SomeSecond Query blk)) "Query"
+    [ rt (Proxy @blk)                         "blk"
+    , rt (Proxy @(GenTx blk))                 "GenTx"
+    , rt (Proxy @(ApplyTxErr blk))            "ApplyTxErr"
+    , rt (Proxy @(SomeSecond BlockQuery blk)) "BlockQuery"
+    , rtWith (Proxy @(SomeSecond Query blk))  "Query"
```

Comment:
`rtWith` seems fine, but so does the pattern in the existing tests. Please make it uniform, either by inlining `rtWith` here and discarding the binding or else change the exists version-sensitive tests to also use `rtWith`.

Edit: The `roundtrip Result` test might not fit with the `rtWith` mold.